sdk/to/to.go

```diff
@@ -41,3 +41,66 @@ func StringPtr(s string) *string {
 func TimePtr(t time.Time) *time.Time {
 	return &t
 }
+
+// ArrayOfInt32Ptr returns an array of *int32 from the specified values.
```

Comment:
SliceOf.... ?

Reply:
I think for being idiomatic slice makes more sense, but I also know that this may be used with fields that have array in the name. I'd be ok with SliceOf

Reply:
You bring up a good point.  I think in our codegen we use the term "array".  If we wanted to use "slice" here, then IMO we should update the codegen to call it "slice" there too.  I will add though that swagger uses the term "array".

Reply:
In our codegen we call these `StringArray`, `Int32Array`, etc.  Perhaps we should rename these to `Int32PtrArray()` etc for consistency?  @JeffreyRichter

Reply:
I do like consistency, so yes, I think so.

Reply:
Fixed